ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -0,0 +1,301 @@
+import * as React from 'react';
+
+import {Component} from '@angular/core';
+
+import {Button, Clickable} from 'app/components/buttons';
+import {FlexColumn, FlexRow} from 'app/components/flex';
+import {ClrIcon} from 'app/components/icons';
+import {TextInput} from 'app/components/inputs';
+import {workspaceAdminApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {ReactWrapperBase} from 'app/utils';
+import {
+  getSelectedPopulations,
+  getSelectedResearchPurposeItems
+} from 'app/utils/research-purpose';
+
+import {
+  AdminWorkspaceResources,
+  UserRole,
+  Workspace
+} from 'generated/fetch';
+
+
+const styles = {
+  fixedWidthWithMargin: {
+    width: '10rem',
+    marginRight: '1rem'
+  },
+  dropdownIcon: {
+    marginRight: '1rem',
+    transform: 'rotate(180deg)',
+    transition: 'transform 0.5s',
+    color: colors.primary
+  },
+  dropdownIconOpen: {
+    transform: 'rotate(0deg)',
+  }
+};
+
+const FlexColumnWithRightMargin = ({style = {}, children}) => {
```

Comment:
[nit] Would it be possible to shorten this name? It distracts a bit from the markup since it's used a lot. Some ideas:
- FlexWithMargin
- AdminFlexColumn
- AdminFlex